website_v2/gcloud-powershell-website-v2/www/static/templates/cmdlet.html

```diff
@@ -0,0 +1,13 @@
+﻿<div class="info">
+    <h1>{{ref.params.cmdlet}}</h1>
```

Comment:
Are you pulling the name straight form the params without any validation?  What if it's invalid?  

This is angular so you shouldn't have to worry about injection issues but better to escape things anyways